cmd/sebak/cmd/run.go

```diff
@@ -44,19 +44,21 @@ var (
 	flagTLSCertFile         string = sebakcommon.GetENVValue("SEBAK_TLS_CERT", "sebak.crt")
 	flagTLSKeyFile          string = sebakcommon.GetENVValue("SEBAK_TLS_KEY", "sebak.key")
 	flagValidators          string = sebakcommon.GetENVValue("SEBAK_VALIDATORS", "")
-	flagThreshold           string = sebakcommon.GetENVValue("SEBAK_THRESHOLD", "66")
+	flagSIGNThreshold       string = sebakcommon.GetENVValue("SEBAK_SIGN_THRESHOLD", "66")
+	flagACCEPTThreshold     string = sebakcommon.GetENVValue("SEBAK_ACCEPT_THRESHOLD", "66")
```

Comment:
Isn't that a revert to the state in master ? Why do we have 2 different thresholds ?

Reply:
https://github.com/bosnet/sebak/pull/240#discussion_r209826426